Lesson 32 HomeWork/Lesson 32 HomeWork/ContentView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ContentView.swift
+//  Lesson 32 HomeWork
+//
+//  Created by Ernest Mihasenko on 28.06.22.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        Text("Hello, world!")
+            .padding()
+            .backGroundModifier()
+            .fontViewModifier()
+            .upperLabelModifier()
+    }
+}
+
+
+struct FontViewModifier: ViewModifier {
+    func body(content: Content) -> some View {
+            content
+                .font(.custom("Marker Felt", size: 50))
+                .foregroundColor(.brown)
+        }
+    
+}
+
+struct BackGroundViewModifier: ViewModifier {
+    func body(content: Content) -> some View {
+        content
+            .background(Color.purple)
+            .cornerRadius(50)
+    }
+}
+
+struct UpperLabelModifier: ViewModifier {
+    var text: String
+    
+    func body(content: Content) -> some View {
+        ZStack(alignment: .top) {
```

Comment:
The task meant to put text above vertically, not on top of the element. So please use `VStack` here